std/file.d

```diff
@@ -1008,8 +1009,11 @@ Delete file `name`.
 Params:
     name = string or range of characters representing the file _name
 
-Throws: $(LREF FileException) on error.
- */
+Throws:
+    $(LREF FileException) on error.
+
+    On Windows it may also give an "Access denied" error, which may require you to call $(LREF setAttributes) beforehand with $(LREF FileAttribute).normal
+*/
```

Comment:
Please do not mix unrelated changes in the same commit / pull request. Remove this change from this pull request, and add it to a separate one.